client/modules/reconportal/src/LeafletMap/ReconPortalPopUp.tsx

```diff
@@ -31,9 +30,13 @@ export const ReconPortalPopup: React.FC<ReconPortalPopuprops> = ({
         </div>
         <Tag
           color={getReconEventColor(dataset)}
-          className={styles.eventDetailTag}
+          style={{
+            fontWeight: 600,
+            fontSize: 14,
+            textTransform: 'capitalize',
+          }}
```

Comment:
Can we tell why `styles.eventDetailTag` is not working? Does it need to be more specific? I think a common style would be good in this case, as this same style set is used in `ReconSidePanel`